src/app/core/models/data-service.model.ts

```diff
@@ -44,13 +44,13 @@ export abstract class DataService<T extends DataType> {
       .valueChanges();
   }
 
-  getById(id: string): Observable<T> {
+  getById(id: string, validateDeletedFlag= true): Observable<T> {
```

Comment:
Something is happening, but the pre-commit hook is not running in your environment, please make sure to install dependencies again